aws/ec2metadata/api.go

```diff
@@ -10,6 +14,30 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/fullsailor/pkcs7"
```

Comment:
I don't think it would be good to pull in an additional dependency into the SDK.

Reply:
the dependency is kind of necessary to parse the pkcs7 structure; there's no built-in package to do this right now.